spec/jobs/get_usps_proofing_results_job_spec.rb

```diff
@@ -1,1901 +1,2644 @@
+                ippAssuranceLevel: '1.5',
+              }
+            end
+
+            let(:response_analytics) do
+              {
+                fraud_suspected: response_body[:fraudSuspected],
+                primary_id_type: response_body[:primaryIdType],
+                secondary_id_type: response_body[:secondaryIdType],
+                failure_reason: response_body[:failureReason],
+                transaction_end_date_time: usps_enrollment_end_date.getlocal('UTC'),
+                transaction_start_date_time: usps_enrollment_start_date.getlocal('UTC'),
+                status: response_body[:status],
+                assurance_level: response_body[:assuranceLevel],
+                proofing_post_office: response_body[:proofingPostOffice],
+                proofing_city: response_body[:proofingCity],
+                proofing_state: response_body[:proofingState],
+                scan_count: response_body[:scanCount],
+                response_message: response_body[:responseMessage],
+                response_present: true,
+              }
+            end
+
+            context 'when the InPersonEnrollment has fraud results pending' do
+              before do
+                allow(IdentityConfig.store).to receive(
+                  :in_person_proofing_enforce_tmx,
+                ).and_return(true)
+                enrollment.profile.update(
+                  fraud_pending_reason: 'threatmetrix_review',
+                  proofing_components: { threatmetrix_review_status: 'review' },
                 )
-
-                job.perform(Time.zone.now)
-
-                pending_enrollment.reload
-                expect(pending_enrollment.enrollment_established_at).to eq(Time.zone.now - 3.days)
-                expect(pending_enrollment.status_updated_at).to eq(Time.zone.now - 1.day)
-                expect(pending_enrollment.status_check_attempted_at).to eq(Time.zone.now)
-                expect(pending_enrollment.status_check_completed_at).to eq(Time.zone.now)
               end
 
-              expect(pending_enrollment.profile.active).to eq(false)
-              expect(pending_enrollment.pending?).to be_truthy
-
-              expect(job_analytics).not_to have_logged_event(
-                'GetUspsProofingResultsJob: Enrollment status updated',
-              )
-
-              expect(job_analytics).to have_logged_event(
-                'GetUspsProofingResultsJob: Enrollment incomplete',
-                hash_including(
-                  response_message: 'Customer has not been to a post office to complete IPP',
-                  job_name: 'GetUspsProofingResultsJob',
-                ),
-              )
-            end
-          end
-
-          context 'when a timeout error occurs' do
-            before(:each) do
-              stub_request_proofing_results_with_timeout_error
-            end
+              context 'when the USPS proofing results has a passing status' do
```

Comment:
In the code, we call `enrollment.profile.deactivate_for_fraud_review` - can we add a spec to ensure profile is deactivated for fraud review?ie check all of the following attributes on profile: 
      active: false,
      fraud_review_pending_at: Time.zone.now,
      fraud_rejection_at: nil,
      in_person_verification_pending_at: nil,

Reply:
I think there is an existing spec for this in the 'when the USPS proofing results has fraud suspected' context block that starts on line 2286?  The context block has an it statement on line 2339
```
                  it "deactivates the enrollment's profile" do
                    expect(enrollment.reload.profile).to have_attributes(
                      active: false,
                      deactivation_reason: 'verification_cancelled',
                      in_person_verification_pending_at: nil,
                    )
                  end
```
The additional attributes Gina mentioned could be tested above.